toolkit/app-write-mram.py

```diff
@@ -196,15 +196,18 @@ def main():
     parser.add_argument(
         "-V", "--version", help="Display Version Number", action="store_true"
     )
+    parser.add_argument("--part", type=str, help="Part Number")
+    parser.add_argument("--rev", type=str, help="Part Revision")
+    parser.add_argument("--jtag", type=str, help="JTAG Interface", default="J-Link")
+    parser.add_argument("--mram", type=str, help="MRAM Interface", default="isp")
```

Comment:
Is it worth putting these common lines in a separate module / helper function, so they don't need to be repeated?

Also, maybe prefix them all by `cfg-` so it's obvious they are a group of options relating to the `global-cfg.db` file?  Eg `--cfg-part` and `--cfg-rev` and `--cfg-jtag` and `--cfg-mram`.

Reply:
> Is it worth putting these common lines in a separate module / helper function

I've added `cfg` prefix, but not sure about the common function. I could add one to `utils/config.py`, but I like seeing all the options in one place, and don't like all the magic functions imported globally everywhere.

Reply:
IMO repeating code is much worse than having to import a single function in three files.

Reply:
You can leave it as-is here.  Can refactor later if we want.

Reply:
Yes, let's move forward with this as is for now.